app/test/socobo-inventory/socobo-inventory-create.html

```diff
@@ -66,7 +67,9 @@
       });
 
       test('create button is hidden when in edit mode', function(done) {
-        dialog.show({});
+        var item = {measurement: {value: 'pcs'}, desc: 'test item', info: 'test info'};
```

Comment:
@flashback2k14  No idea why but it didn't work for me with the latest state. You could also check this again and if I'm wrong just close my pull request, please.